### PR TITLE
Don't inlcude 'you' in any rule descriptions

### PR DESCRIPTION
**Pull request by mlinksva**
Descriptions should make sense whether reader is licensor or licensee.

Pointed out by @HawkonDK at https://github.com/github/choosealicense.com/issues/499#issuecomment-289321921

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,25 +80,25 @@ The license properties (rules) are stored as a bulleted list within the licenses
...
 
 #### Conditions
 
-* `include-copyright` - Include a copy of the license and copyright notice with the software.
-* `document-changes` - Indicate changes made to the code.
-* `disclose-source` - Source code must be made available when distributing the software.
-* `network-use-disclose` - Users who interact with the software via network are given the right to receive a copy of the corresponding source code.
+* `include-copyright` - A copy of the license and copyright notice must be included with the software.
+* `document-changes` - Changes made to the code must be documented.
+* `disclose-source` - Source code must be made available the software is distributed.
```
Comment (waldyrious) [inline README.md]: Sentence is incomplete here ("...available the software...")

File: _data/rules.yml [hunk 2]
```diff
@@ -5,27 +5,27 @@ permissions:
...
-- description: Include a copy of the license and copyright notice with the software.
+- description: A copy of the license and copyright notice must be included with the software.
   label: License and copyright notice
   tag: include-copyright
-- description: Indicate changes made to the code.
+- description: Changes made to the code must be documented.
   label: State changes
   tag: document-changes
-- description: Source code must be made available when distributing the software.
+- description: Source code must be made available the software is distributed.
```
Comment (waldyrious) [inline _data/rules.yml]: Incomplete sentence here as well.

File: README.md [hunk 3]
```diff
@@ -80,25 +80,25 @@ The license properties (rules) are stored as a bulleted list within the licenses
 
 * `commercial-use` - This software and derivatives may be used for commercial purposes.
 * `modifications` - This software may be modified.
-* `distribution` - You may distribute this software.
-* `private-use` - You may use and modify the software without distributing it.
-* `patent-use` - This license provides an express grant of patent rights from the contributor to the recipient.
+* `distribution` - This software may be distributed.
+* `private-use` - This software may be used and modifed in private.
```
Comment (benbalter) [inline README.md]: > This software may be used and modifed without distributing it?

Also `modified`

**Review by benbalter — APPROVED** [review]
The passive voice is killing me, but really glad you were able to do this without introducing "licensor" and "licensee".

**Comment by mlinksva** [comment]
Thanks. I feel terrible about the passive voice. Thought about making "anyone" or "the public" or "all" the subject in each rule description, but that gets very tedious. Better ideas welcome, anytime.

**Pull request merged**